test/transaction_test_local.dart

```diff
@@ -1,15 +1,19 @@
 import 'package:flutter/foundation.dart';
-import 'package:flutter_app/api_calls/wallet_api.dart';
+import 'package:flutter_app/api_calls/cosmos_api.dart';
 import 'package:flutter_app/global.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
```

Comment:
lets rename the file to `local_transaction_test.dart`

Reply:
✅